app/javascript/components/dropdowns/track-menu/ActivateLearningModeModal.tsx

```diff
@@ -52,7 +53,7 @@ export const ActivateLearningModeModal = ({
       <h2>Activate Learning Mode</h2>
       <p>
         Activating Learning Mode will enable Concepts and Learning Exercises on
-        this track, but will lock Exercises that you've not yet complete
+        this track, but will lock Exercises that you&apos;ve not yet complete
```

Comment:
This is unrelated, so you could consider a separate PR fixing just these :) It doesn't distract too much from the PR though in this case, but just as a general suggestion ;)